RNLiveMarkdown.podspec

```
@@ -24,15 +32,23 @@ Pod::Spec.new do |s|
 
   s.source_files = "apple/**/*.{h,m,mm}", "cpp/**/*.{h,cpp}"
 
-  s.dependency "RNReanimated/worklets"
+  if worklets_installed
+    s.dependency "RNWorklets"
+  else
+    s.dependency "RNReanimated/worklets"
+  end
 
-  s.xcconfig = {
+  xcconfig = {
     "OTHER_CFLAGS" => "$(inherited) -DREACT_NATIVE_MINOR_VERSION=#{react_native_minor_version}",
     "HEADER_SEARCH_PATHS" => [
-      "\"$(PODS_ROOT)/#{react_native_reanimated_node_modules_dir_from_pods_root}/apple\"",
-      "\"$(PODS_ROOT)/#{react_native_reanimated_node_modules_dir_from_pods_root}/Common/cpp\"",
+      "\"$(PODS_ROOT)/#{react_native_worklets_node_modules_dir_from_pods_root}/apple\"",
+      "\"$(PODS_ROOT)/#{react_native_worklets_node_modules_dir_from_pods_root}/Common/cpp\"",
     ].join(' '),
   }
+  if worklets_installed
+    xcconfig["OTHER_CFLAGS"] << " -DWORKLETS_INSTALLED=1"
```

On Android we don't define the value
```suggestion
    xcconfig["OTHER_CFLAGS"] << " -DWORKLETS_INSTALLED"
```